### PR TITLE
Adds flake8-copyright checker 

### LINKED ISSUE #310: Add flake8 plugins
- [ ] flake8 plugin to check that all function parameters have annotation
  - [ ] components: #377
  - [x] cryptography #388 
  - [x] databases #380 
  - [ ] federated
    - [ ] data #390 
    - [ ] plan #391
    - [ ] task #393 
  - [ ] interface
  - [ ] native
  - [ ] pipelines
  - [ ] plugins
  - [ ] protocols
  - [ ] transport
  - [ ] utilities
  - [ ] openfl-tutorials (optional) #383
  - [x] openfl-workspace (optional) #381 
  - [ ] final check
- [x] flake8 plugin to check that all files have correct copyrights: #370 
- [ ] flake8 plugin to check trailing comma [flake8-commas](https://pypi.org/project/flake8-commas/)

### PR DESCRIPTION
**Pull request by ViktoriiaRomanova**
* Added flake8-copyright plugin which checks for copyright notices in all python files
* Fixed errors

### CODE REVIEW

File: requirements-linters.txt [hunk 1]
```diff
@@ -3,6 +3,7 @@ flake8-broken-line
 flake8-bugbear
 flake8-builtins
 flake8-comprehensions
+flake8-copyright
```
Comment (dmitryagapov) [inline requirements-linters.txt:6]: Does it validate all header's lines?
```
# Copyright (C) 2021-2022 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```
Maybe we should chenge default regex?

**Review by dmitryagapov — CHANGES_REQUESTED** [review]

File: requirements-linters.txt (hunk 1 above)
Comment (ViktoriiaRomanova) [reply requirements-linters.txt:6]: Why? Default regex describe that part: ` Copyright (C) 2021-2022 `. We should check something else?

Comment (dmitryagapov) [reply requirements-linters.txt:6]: It goesn't check `SPDX-License-Identifier: Apache-2.0`

**Comment by ViktoriiaRomanova** [comment]
Jenkins please retry a build

**Comment by ViktoriiaRomanova** [comment]
Jenkins please retry a build

File: openfl-workspace/keras_nlp/src/dataloader_utils.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
-"""Copyright (C) 2020-2021 Intel Corporation.
+"""Copyright (C) 2020-2021 Intel Corporation
+   SPDX-License-Identifier: Apache-2.0
 
 Licensed subject to the terms of the separately executed evaluation
 license agreement between Intel Corporation and you.
```
Comment (itrushkin) [inline openfl-workspace/keras_nlp/src/dataloader_utils.py:5]: Should we include full license text in checker rules in order to use only one type of file header?

File: setup.cfg [hunk 3]
```diff
@@ -13,3 +15,4 @@ avoid-escape = False
 import-order-style = smarkets
 application-import-names = openfl
 ignore-names=X_*,X,X1,X2
+copyright-regexp = Copyright\s+(\(C\)\s+)?\d{4}([-,]\d{4})*\s+Intel\sCorporation(\s)*\n\s*#*\sSPDX-License-Identifier:\sApache-2.0
```
Comment (dmitryagapov) [inline setup.cfg:18]: Could you please put `copyright-regexp` and `copyright-check` together.

**Review by dmitryagapov — CHANGES_REQUESTED** [review]

File: openfl-workspace/keras_nlp/src/dataloader_utils.py (hunk 2 above)
Comment (dmitryagapov) [reply openfl-workspace/keras_nlp/src/dataloader_utils.py:5]: @alexey-gruzdev could you please clarify this?

Comment (alexey-gruzdev) [reply openfl-workspace/keras_nlp/src/dataloader_utils.py:5]: @itrushkin  @dmitryagapov @ViktoriiaRomanova it's okay to keep only license header, not a full text.

**Comment by ViktoriiaRomanova** [comment]
Jenkins please retry a build

**Review by dmitryagapov — APPROVED** [review]

**Review by aleksandr-mokrov — APPROVED** [review]

**Review by alexey-khorkin — APPROVED** [review]

**Review by alexey-gruzdev — APPROVED** [review]

**Pull request merged**